proto/internal/temporal/server/api/enums/v1/cluster.proto

```diff
@@ -38,4 +38,5 @@ enum HealthState {
     HEALTH_STATE_UNSPECIFIED = 0;
     HEALTH_STATE_SERVING = 1;
     HEALTH_STATE_NOT_SERVING = 2;
+    HEALTH_STATE_STARTING = 3; 
```

Comment:
not sure if the starting make sense or just use UNKNOWN?

Reply:
I could change this to NOT_READY? I suppose this could also be flipped "off" by the service if the host/pod is no longer healthy re: our conversation earlier that we don't just set this on startup

Reply:
Changed!